snap/snap-in-docker.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+#
+# Build the ipfs-cluster snaps and push them to the store using a docker container.
+
+set -ev
+
+docker run -v $(pwd):$(pwd) -t snapcore/snapcraft sh -c "apt update -qq && cd $(pwd) && ./snap/snap-multiarch.sh"
```

Comment:
I think you can just call this line directly in `.travis.yml`

Reply:
You are right, thanks for catching that.